03 Misc/08 DefinePlugin advanced/prod.webpack.config.js

```diff
@@ -68,4 +69,4 @@ module.exports = function () {
       })
     ],
   });
-}
+})();
```

Comment:
Nice, what do you think if we remove `function` stuff and export something like:

```
module.exports = webpackMerge(commonConfig, {
...
})
```